registry/src/pages/register.js

```diff
@@ -29,6 +29,7 @@ const Register = () => {
     if (isAuthenticated) {
       setCookie("uuid", uuid);
       navigate("/manage/projects");
+      window.location.reload();
```

Comment:
Is it really required ? 
I think it might not be the good idea of using this as it is basically reloading the whole app that might cause optimisation issues in future. 

Why do you want to reload the page before navigating ? @henilp105

Reply:
@arteevraina This reload is to handle the state management issues , it runs only after the navigating , it is important to reload the state , we can avoid this but it may later cause bug in refreshing the state variables ( its sort of a safety patch to avoid un authenticated access).